### PR TITLE
update commands to display hello-5.ko console messages

### PR DESCRIPTION
**Pull request by marconi1964**
Shell commands of "sudo insmod hello-5.ko" and "sudo rmmod hello-5" will not display the messages on console. Instead, the pr_info messages shall be displayed with dmesg or journalctl commands only. So update the commands accordingly. To answer to [issue#111](https://github.com/sysprog21/lkmpg/issues/111#issue-1002892953)

### CODE REVIEW

File: lkmpg.tex [hunk 1]
```diff
@@ -412,31 +412,35 @@ \subsection{Passing Command Line Arguments to a Module}
...
 $ sudo insmod hello-5.ko mystring="bebop" myintarray=-1
-myshort is a short integer: 1
-myint is an integer: 420
-mylong is a long integer: 9999
-mystring is a string: bebop
-myintarray[0] = -1
-myintarray[1] = 420
-got 1 arguments for myintarray.
+$ dmesg | tail -7
+[191207.935945] myshort is a short integer: 1
```
Comment (fennecJ) [inline lkmpg.tex:416]: Do not leave the kernel’s timestamps here, because it varies over time.

Comment (marconi1964) [reply lkmpg.tex]: Time tag removed.

**Comment by fennecJ** [comment]
Rebase the branch into one commit, also check the git message rules:
https://chris.beams.io/posts/git-commit/
> Limit the subject line to 50 characters

> Capitalize the subject line

Fix those minor problems and I think it is well-prepared for merge.

**Review by jserv — CHANGES_REQUESTED** [review]
Append "Close #111" at the end of git commit message.

File: lkmpg.tex (hunk 1 above)
Comment (P2pig) [reply lkmpg.tex]: $ dmesg -t | tail -f would easily get kernel output without timestamps and data appended data as file grows. :D

**Comment by jserv** [comment]
@marconi1964, use `git rebase` rather than `git merge`.

**Comment by fennecJ** [comment]
@marconi1964 
Check the following guide to see how to squash several commits into one commit
https://www.internalpointers.com/post/squash-commits-into-one-git

By the way, you can just use `git commit --amend` to edit the most recent commit message only.

**Comment by jserv** [comment]
Thank @marconi1964 for contributing!

**Comment by marconi1964** [comment]
Not sure if I did it correctly of `git rebase`. Just document it as below. 
It is clear to me to update the commit subject, with clear guidance provided. Thank you @fennecJ.

Consulted my enterprise server firmware friend for help on rebase. Surprisingly, such command is not often used, due to the code/modules are usually maintained by a single person, instead of collaborative works. Could only counting on Google, which was not clear between github remote on sysprog21/lkmpg, on marconi1964/lkmpg, and local repo lkmpg.

Issues I faced were
- "Git: "Cannot 'squash' without a previous commit" error while rebase" Not sure why git insist to 'pick' on first commit?
- struggled on several `git rebase` tries
```
# after several try and error on the following commands, 
$ git remote add lkmpg https://github.com/sysprog21/lkmpg.git
$ git fetch lkmpg
$ git merge-base master lkmpg/master
$ git rebase -i 1a6fb67cf292f4a0f2191e7f8720afa678d787b2
# eventually, I used this one which might not be the correct way, as I combined my commits with someone else's.
$ git $ git rebase -i HEAD~7
```
- `git push` rejected 
```
$ git push
To github.com:marconi1964/lkmpg.git
 ! [rejected]        master -> master (non-fast-forward)
```

Eventually, 'succeeded' (kind of) with the following commands
```
$ git clone git@github.com:marconi1964/lkmpg.git
$ cd lkmpg
$ git add lkmpg.tex          # adding -t after dmesg
$ git commit -m "Add cmds to disp hello-5.ko msg on console, close #111"
$ git rebase -i HEAD~7
$ git pull --rebase
$ git push
```

**Comment by fennecJ** [comment]
The reason why you're rejected while trying to push to Github is that after you rebase a branch,
your local branch becomes different from the remote one and causes conflict.
To solve that, you need to push with the force option `git push -f`.
The solution you mentioned at the end: `git pull --rebase` means to stash your local change, resetting the branch's
state the last time you pull from the remote repo and apply remote change into local repo; then add back your local change
into the branch without conflict. Which makes your previous squash reverted.

**Pull request merged**